unified-runtime/source/adapters/hip/image.cpp

```diff
@@ -902,12 +949,13 @@ UR_APIEXPORT ur_result_t UR_APICALL urBindlessImagesImageCopyExp(
         cpy_desc.Height =
             std::max(MinCopyHeight, pCopyRegion->copyExtent.height);
         cpy_desc.Depth = pCopyRegion->copyExtent.depth;
-        // 'hipMemcpy3DAsync' requires us to correctly create 'hipMemcpy3DParms'
-        // struct object which adds a little complexity (e.g. 'hipPitchedPtr').
+        // 'hipMemcpy3DAsync' requires us to correctly create
+        // 'hipMemcpy3DParms' struct object which adds a little complexity
+        // (e.g. 'hipPitchedPtr').
         UR_CHECK_ERROR(hipDrvMemcpy3DAsync(&cpy_desc, Stream));
       }
       // Synchronization is required here to handle the case of copying data
-      // from host to device, then device to device and finally device to host.
+      // from host to device, then device to device and finally device to host
```

Comment:
Nit: not sure why the full stop was removed here?

```suggestion
      // from host to device, then device to device and finally device to host.
```

Similarly, the formatted lines above seem to fit within 80 columns, but if clang-format prefers the change then that's fine.

Reply:
Yeah not sure how that happened. I'll restore the original.

Reply:
Updated.